pkg/web/templates/partials/counterparty/counterparty_list.html

```diff
@@ -18,8 +17,7 @@
         <td>{{ .Name }}</td>
         <td>{{ .Website }}</td>
         <td>{{ .Protocol }}</td>
-        <td class="px-16">-</td>
-        <td class="px-16">-</td>
+        <td>{{ .TravelAddress }}</td>
```

Comment:
@daniellemaxwell hopefully this will work now!

Reply:
It does. Thanks! I made a minor CSS update so that the travel address wraps in the table.